src/components/Dashboard/TodoList.vue

```diff
@@ -1128,12 +1129,6 @@ export default class TodoList extends Mixins(
         this.setCurrentFilingStatus(FilingStatus.NEW)
         this.$router.push({ name: Routes.ANNUAL_REPORT, params: { filingId: '0' } }) // 0 means "new AR"
         break
-      // FUTURE: uncomment when/if we have NRs without a draft
```

Comment:
I don't think we're ever going back to loading this UI with just a NR (and no draft), so I stripped out this code.